packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -271,6 +271,30 @@ export class CWithdrawFromVault extends ComposableBuffer<WithdrawFromVault> {
   }
 }
 
+/**
+ * CloseVault DeFi Transaction
+ */
+export interface CloseVault {
```

Comment:
I think what @surangap mean in the original comment is to move these into `dftx/vault`

Can refer to https://github.com/DeFiCh/ain/tree/master/src/masternodes (which IMO, some of them are not quite in the right place)

Reply:
For reference, what are the things that you think are not in the right place, and where do you think they should be?